.babelrc

```diff
@@ -5,7 +5,7 @@
       "@babel/preset-env",
       {
         "corejs": 3,
-        "modules": false, // `false` will not transform ES6 module syntax
+        "modules": "commonjs", // `false` will not transform ES6 module syntax
```

Comment:
See this issue: https://github.com/noppa/ng-hot-reload/issues/31

Reply:
Should the comment be updated and potentially link to that issue?

Reply:
Yes, it should. Good catch. I'll update it.